geometry_msgs/Point[] obstalces_position
float64[] obstacles_radius